packages/reakit/src/Tabs/TabsTab.md

```diff
@@ -1,5 +1,5 @@
 ```jsx
-import { Block } from "reakit";
+import { Tabs } from "reakit";
```

Comment:
Hi, @ramiel. Thanks for your contribution.

That example also uses `Block`, so I think it should be:

```suggestion
import { Tabs, Block } from "reakit";
```